ajax/ajax.js

```diff
@@ -0,0 +1,5 @@
+const xhttp = new XMLHttpRequest(), 
+  log = console.log;
```

Comment:
no need to assign javascript inbuild method to variable

Reply:
Please tell alternative solution